crates/model/src/solver_competition.rs

```diff
@@ -5,26 +5,37 @@ use crate::{
 use primitive_types::{H160, H256, U256};
 use serde::{Deserialize, Serialize};
 use serde_with::serde_as;
-use std::collections::HashMap;
+use std::collections::BTreeMap;
 
 #[derive(Clone, Debug, Default, Deserialize, Serialize, PartialEq)]
 #[serde(rename_all = "camelCase")]
 pub struct SolverCompetitionResponse {
     pub gas_price: f64,
+    pub auction_start_block: u64,
     pub liquidity_collected_block: u64,
     pub competition_simulation_block: u64,
     pub transaction_hash: Option<H256>,
+    pub auction: CompetitionAuction,
     pub solutions: Vec<SolverSettlement>,
 }
 
+#[serde_as]
+#[derive(Clone, Debug, Default, Deserialize, Serialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct CompetitionAuction {
+    pub orders: Vec<OrderUid>,
+    #[serde_as(as = "BTreeMap<_, DecimalU256>")]
+    pub prices: BTreeMap<H160, U256>,
+}
+
 #[serde_as]
 #[derive(Clone, Debug, Default, Deserialize, Serialize, PartialEq)]
 #[serde(rename_all = "camelCase")]
 pub struct SolverSettlement {
     pub solver: String,
     pub objective: Objective,
-    #[serde_as(as = "HashMap<_, DecimalU256>")]
-    pub prices: HashMap<H160, U256>,
+    #[serde_as(as = "BTreeMap<_, DecimalU256>")]
+    pub clearing_prices: BTreeMap<H160, U256>,
```

Comment:
Just renamed to make the distinction between auction prices and clearing prices more visible. Also, the `BTreeMap` so that we have ordered token/price pairs.